ci/package.sh

```diff
@@ -22,7 +22,7 @@ echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 docker build -t bookingcom/shipper:$TRAVIS_COMMIT -f Dockerfile.shipper .
 docker push bookingcom/shipper:$TRAVIS_COMMIT
 
-docker build -t bookingcom/shipper-state-metrics:$TRAVIS_COMMIT -f Dockerfile.metrics .
```

Comment:
This file is used by internal CI.

Reply:
I'll keep this one, since having the full name of the binary it holds is a good thing for discovery. Will handle the issues in the internal CI once this work has been merged.